table/table_builder.h

```diff
@@ -37,12 +37,14 @@ struct TableReaderOptions {
                      bool _skip_filters = false, bool _immortal = false,
                      bool _force_direct_prefetch = false, int _level = -1,
                      BlockCacheTracer* const _block_cache_tracer = nullptr,
-                     size_t _max_file_size_for_l0_meta_pin = 0)
-      : TableReaderOptions(_ioptions, _prefix_extractor, _env_options,
-                           _internal_comparator, _skip_filters, _immortal,
-                           _force_direct_prefetch, _level,
-                           0 /* _largest_seqno */, _block_cache_tracer,
-                           _max_file_size_for_l0_meta_pin) {}
+                     size_t _max_file_size_for_l0_meta_pin = 0,
+                     const std::string& _db_session_id = "",
```

Comment:
When we get to a context associated with a single table file (such as this whole file), we should call it `cur_db_session_id` rather than `db_session_id` because it's the id of the running DB, not the id in the table file properties from the DB that generated it, which might be different.

Reply:
Sure, will update it where it is associated with the table file.